br/pkg/lightning/backend/external/util.go

```diff
@@ -118,3 +121,39 @@ func GetAllFileNames(
 	sort.Strings(stats)
 	return data, stats, nil
 }
+
+// CleanUpFiles delete all data and stat files under one subDir.
+func CleanUpFiles(ctx context.Context,
+	store storage.ExternalStorage,
+	subDir string,
+	concurrency uint) error {
+	dataNames, statNames, err := GetAllFileNames(ctx, store, subDir)
+	if err != nil {
+		return err
+	}
+
+	eg, _ := errgroup.WithContext(ctx)
```

Comment:
if we don't need to check the error from eg.Go, we can use the zero value of errgroup.Group